controller/translationcontroller.php

```diff
@@ -95,18 +95,18 @@ public function getLanguageStrings() {
 			'revision.restored' => $this->trans->t('Revision restored'),
 
 			// js/app/controllers/settings.js
-			'bookmarklet.text' => $this->trans->t('Save in passman'),
+			'bookmarklet.text' => $this->trans->t('Save in Passman'),
```

Comment:
Hmm just saw this thread is at the wrong line. 
Anyway's 'Javascript Bookmark' doesn't cover what it does, bookmarklet does.